LabDrivers/Tool.py

```diff
@@ -535,6 +535,29 @@ def change_interface(self, intf, **kwargs): # only really applicable to prologix
         else:
             logging.debug("Change interface called with %s, current interface is %s"%(intf, self.interface))
 
+
+    def set_ren(self, level):
+        # this function is used as an alias for control_ren, which is built in to pyvisa controller, and programmed into Prologix
+        mode = int(level)
+        if self.interface == INTF_SERIAL:
+            self.serial_set_ren(level)
+        if hasattr(self.connection, 'control_ren'):
+            if self.interface == INTF_PROLOGIX:
+                self.connection.control_ren(mode, self.resource_number)
+            else:
+                self.connection.control_ren(mode)
+        else:
+            print("Reminder to finish this")
+
+    def serial_set_ren(self, level):
+        if level in [1, 3]:
+            self.connection.write("REN")
+
+        if level in [2,6]:
```

Comment:
why not using `else if` as well?

Reply:
I did not use any else if's in the function, as there is overlap for REN levels. In this specific case there isn't, however, so I will implement that change before merging.